index.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
+  configs: {
+    tslint: require('./lib/config/all.js'),
```

Comment:
What would this look like for the consumer extending this config? Would the have to `extend: 'tslint-plugin-shopify/tslint'`?